src/index.js

```diff
@@ -68,7 +68,7 @@ class TelldusTDToolPlatform {
   }
 }
 
-/*
+/**
```

Comment:
The difference in that case would be that other comments using double-star in the beginning is meant for documenting functions, whereas this only documents what happens in the following lines, not the interface that could be interesting for consumers.

However, I don't mind keeping this anyway :)